cmds/comm/comm_test.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
As pointed by Floren in another review, the file needs the license notice, doesn't it?